Hellfire SPMA: Various Updates

HellDragon-HK opened the pull request:
Found in the UT3 Editor's SPMA sounds, this is either very quiet in-game or unused

HellDragon-HK committed:
Hellfire SPMA: Turret Rotate Sound Addition

Found in the UT3 Editor's SPMA sounds, this is either very quiet in-game or unused

GreatEmerald commented:
Hm, I'm not seeing this in the code. There is a sound for rotating the main cannon, however (`A_Vehicle_SPMA.SoundCues.A_Vehicle_SPMA_CannonRotate`), is that the same sound?

HellDragon-HK commented:
No it's not, this is something that was I guess unused or used to be used.

GreatEmerald commented:
I'm guessing that it was unused, then. In which case I don't think there's a good reason to reintroduce it if it doesn't really improve anything.

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update UT3HellfireSPMACamera.uc

HellDragon-HK committed:
Update UT3HellfireSPMACannon.uc

HellDragon-HK committed:
Update UT3HellfireSPMAIncomingSound.uc

HellDragon-HK committed:
Update UT3HellfireSPMAShell.uc

HellDragon-HK committed:
Update UT3HellfireSPMAShellChild.uc

HellDragon-HK committed:
Update UT3HellfireSPMASideGun.uc

HellDragon-HK commented:
I think everything is working now including the camera's ambient sound...though possibly a little loud, the engine on the other hand is the opposite and like usual is probably going to be too quiet.

HellDragon-HK committed:
Update Binaries.csv

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update UT3HellfireSPMACamera.uc

HellDragon-HK commented:
Argh this Travis thing hates me, I just uploaded that binary and got the checksum from the same file so how did it fail, a glitch?

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update Binaries.csv

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update UT3HellfireSPMACamera.uc

HellDragon-HK committed:
Update UT3HellfireSPMACannon.uc

HellDragon-HK committed:
Update UT3HellfireSPMAIncomingSound.uc

HellDragon-HK committed:
Update UT3HellfireSPMAShell.uc

Are these sounds backward?

HellDragon-HK committed:
Update UT3HellfireSPMAShellChild.uc

HellDragon-HK committed:
Update UT3HellfireSPMASideGun.uc

Projectile has same problem I had to fix on Hellbender about not being able to do combos so that needs to be looked into

HellDragon-HK commented:
Full update of non-recorded varied sounds in Basic or Extended flavor

Note: Haven't done Goliath yet, it's next on the list so when it's done the impact sound here needs to be changed to use that

Also need to look at fixing the no shock combos issue that I had with the Hellbender

HellDragon-HK committed:
Update UT3HellfireSPMASideGun.uc

HellDragon-HK commented:
Fixed the No Combos problem

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Merge branch 'master' into patch-7

HellDragon-HK commented:
Added suspensions fix to get SPMA out of the ground and reorganized defaults into an imo much more readable list with clear identity of entries....categorization may need some further adjusting though.....still a trial run so to speak.

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update UT3HellfireSPMACamera.uc

HellDragon-HK committed:
Update UT3HellfireSPMACannon.uc

HellDragon-HK committed:
Update UT3HellfireSPMAIncomingSound.uc

HellDragon-HK committed:
Update UT3HellfireSPMAShell.uc

HellDragon-HK committed:
Update UT3HellfireSPMAShellChild.uc

HellDragon-HK committed:
Update UT3HellfireSPMASideGun.uc

HellDragon-HK committed:
Update UT3HellfireSPMA.uc

HellDragon-HK committed:
Update UT3HellfireSPMACannon.uc

HellDragon-HK committed:
Update UT3HellfireSPMAShell.uc

HellDragon-HK committed:
Update UT3HellfireSPMA.uc
